client/src/components/Users/Auth/SignUpCitizen/StepOne.tsx

```diff
@@ -0,0 +1,268 @@
+    const formValues = Object.entries(formData).reduce((acc, [key, val]) => {
+      // @ts-ignore
+      acc[key] = val.value;
+      return acc;
+    }, {});
+    handleNext(formValues);
+  };
+
+  return (
+    <>
+      <Box sx={{ height: '100%', width: '100%' }}>
+        <Typography
+          sx={{ color: '#674E67' }}
+          className={classes.header}
+          lineHeight="87px"
+          fontSize="58px"
+          component="h1"
+          align="left"
+          variant="h4"
+          gutterBottom
+        >
+          Let's get started
+        </Typography>
+        <Typography
+          sx={{ color: '#323232' }}
+          className={classes.subHeader}
+          lineHeight="1.4"
+          fontSize="22px"
+          align="left"
+        >
+          Basic Information
+        </Typography>
+        <Typography
+          sx={{ color: '#323232' }}
+          className={classes.text}
+          lineHeight="1.4"
+          fontSize="16px"
+          align="left"
+          gutterBottom
+        >
+          {/* TODO: Add link to "Login", this will open the login modal */}
+          Already have an account?
+        </Typography>
+        <Box display={'flex'} flexDirection={'row'} width={'100%'}>
+          <Box marginRight={'20px'}>
+            <NameInput
+              id="firstName"
+              name="firstName"
+              label="First Name"
+              placeholder="Jane"
+              onBlur={handleBlur}
+              onChange={handleChange}
+              value={formData.firstName.value}
+              error={formData.firstName.error}
+            />
+          </Box>
+          <Box width={'100%'}>
+            <NameInput
+              id="lastName"
+              name="lastName"
+              label="Last Name"
+              placeholder="Dosis"
+              onBlur={handleBlur}
+              onChange={handleChange}
+              value={formData.lastName.value}
+              error={formData.lastName.error}
+            />
+          </Box>
+        </Box>
+        <EmailInput
+          showStartAdornment={true}
+          onBlur={handleBlur}
+          onChange={handleChange}
+          value={formData.email.value}
+          error={formData.email.error}
+          placeholder="jane@citizen.com"
+        />
+        <PasswordInput
+          id="password"
+          name="password"
+          onBlur={handleBlur}
+          value={formData.password.value}
+          showStartAdornment={true}
+          onChange={handleChange}
+          error={formData.password.error}
+          required
+        />
+        <PasswordInput
+          id="passwordConfirm"
+          name="passwordConfirm"
+          label="Confirm Password"
+          onBlur={handleBlur}
+          value={formData.passwordConfirm.value}
+          showStartAdornment={true}
+          onChange={handleChange}
+          error={formData.passwordConfirm.error}
+          required
+        />
+        <FormControlLabel
+          style={{
```

Comment:
If `sx={..` work for this MUI component, we should use it instead of style, I think.